README.md

```diff
@@ -42,6 +42,26 @@ Include `mh-opsworks-recipes` in your node's `run_list`:
 
 * Dan Collis-Puro - [djcp](https://github.com/djcp)
 
+## Development
+
+* Create a branch off the develop branch
+* Do your work and test it in a running mh-opsworks cluster
+* Be sure to update the "To be released" section of the CHANGELOG.md file with
+  a description of your change, and whether or not it requires a manual recipe
+  invocation. If your change runs as part of the deploy lifecycle, it will be run
+  the next time the main app is deployed to a cluster and therefore does not
+  require a manual chef recipe invocation.
+* Rebase against develop and submit a pull request off of develop.
+
+## Releasing
+
+* Update CHANGELOG.md to collapse all "To be released" changes under a new
+  version, following [semver](http://semver.org).
+* Rebase develop onto master.
+* Merge develop into master.
```

Comment:
I have been thinking that provisioning changes are QA'd a couple ways:

1) If you're working on a feature that requires provisioning changes, they'd be QA'd as part of that feature.
2) When we're testing an RC, that RC would use the "develop" chef recipes branch and they'd be QA'd there.

There is some art to this, of course. There may be situations where we think a provisioning change not tied to a feature is potentially problematic and requires more QA attention, then we might focus on that a bit more.